backend/main.go

```diff
@@ -334,16 +414,19 @@ func (s *Server) handleImprove(w http.ResponseWriter, r *http.Request) {
 func main() {
 	var useInMemStore bool
 	var authMode string
-	var authDomain string
+	var defaultDomain string
 	flag.BoolVar(&useInMemStore, "inmemstore", false, "Use in-memory datastore")
+	flag.StringVar(&defaultDomain, "defaultdomain", "none", "When using inmemstore: the domain that all team permissions are defaulted to")
 	flag.StringVar(&authMode, "authmode", "none", "Set authentication mode, either 'none' or 'firebase'")
-	flag.StringVar(&authDomain, "authdomain", "google.com", "The email domain with read and write access")
 	flag.Parse()
 
 	var store storage.StorageService
 	if useInMemStore {
 		log.Printf("Using in-memory store per command-line flag")
-		store = in_memory_storage.MakeInMemStore()
+		if defaultDomain == "none" {
```

Comment:
Why not just set the default to `google.com`?

Reply:
I'm sure I had a reason why I wrote these lines in the first place but I can't see them being useful at all now, so I've changed the default to google.com now